scripts/install-latest.sh

```diff
@@ -106,9 +106,20 @@ yarn_install() {
   printf "${white}Installing Yarn!$reset\n"
 
   if [ -d "$HOME/.yarn" ]; then
-    printf "$red> ~/.yarn already exists, possibly from a past Yarn install.$reset\n"
-    printf "$red> Remove it (rm -rf ~/.yarn) and run this script again.$reset\n"
-    exit 0
+    if [ -n `which yarn` ]; then
+      LATEST_VERSION=`curl https://yarnpkg.com/latest-version`
```

Comment:
This may need to handle the `--nightly` flag. We have this in `yarn_get_tarball`:

```
  if [ "$1" = '--nightly' ]; then
    url=https://nightly.yarnpkg.com/latest.tar.gz
  else
    url=https://yarnpkg.com/latest.tar.gz
  fi
```

You can use https://nightly.yarnpkg.com/latest-tar-version to get the latest nightly version.